packages/launchpad/src/migration/OptOutModalStep1.vue

```diff
@@ -2,6 +2,7 @@
   <StandardModal
     :model-value="true"
     :title="t('migration.renameAuto.modals.title')"
+    @close="emit('cancel')"
```

Comment:
Why not use the standard event?

```suggestion
    @update:modelValue="emit('cancel')"
```